user_form_app/lib/app/feature/user_form/widgets/gender_widget.dart

```diff
@@ -13,9 +14,19 @@ class GenderWidget extends StatelessWidget {
         Text("Gender", style: Theme.of(context).textTheme.bodyLarge),
         Row(
           children: [
-            Radio(value: 0, groupValue: 0, onChanged: (value) {}),
+            Radio(
```

Comment:
can we create AppRadioWidget?
so that it will be reused in the app and if we need to change anything in the radio button it will be on the one place only :-)